sun-asterisk-research/table_reconstruction
# feat: add a module for detecting table region

## Description

add a module for detecting table region

[#4 (closed)](https://github.com/sun-asterisk-research/table_reconstruction/issues/4 "Add a module for detecting table region") (issue)

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?
- Create and  activate new python environment, after that install all dependencies via command `pip3 install -r requirements.txt`
-  Download 2 files [table_file.pdf](https://drive.google.com/file/d/1ZsFKeUNd5qNngTzl_spLVympa2Xyn40f/view?usp=sharing) and [table_detection.ipynb](https://drive.google.com/file/d/1wjkGgUIn6UAFdGbnKURs2uXvNED8ORVN/view?usp=sharing), put them in the root directory of the repo.
- run Jupyter notebook by using the command `jupyter notebook`. then open **table_detection.ipynb** and run all cells.
Check results of the module at the last cell of the notebook.
[](https://i.imgur.com/HZUCJMV.png)


## Checklist

- [x] I have successfully run the modified code in my local environment with no errors
- [ ] I ran the proper linters and fixed the issues pointed out to make sense
- [ ] I updated the unit test based on my edit
